fusequery/query/src/interpreters/interpreter_describe_table.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020-2021 The Datafuse Authors.
+//
+// SPDX-License-Identifier: Apache-2.0.
+
+use std::sync::Arc;
+
+use common_datablocks::DataBlock;
+use common_datavalues::prelude::*;
+use common_datavalues::series::Series;
+use common_exception::Result;
+use common_planners::DescribeTablePlan;
+use common_streams::DataBlockStream;
+use common_streams::SendableDataBlockStream;
+
+use crate::interpreters::Interpreter;
+use crate::interpreters::InterpreterPtr;
+use crate::sessions::FuseQueryContextRef;
+
+pub struct DescribeTableInterpreter {
+    ctx: FuseQueryContextRef,
+    plan: DescribeTablePlan,
+}
+
+impl DescribeTableInterpreter {
+    pub fn try_create(ctx: FuseQueryContextRef, plan: DescribeTablePlan) -> Result<InterpreterPtr> {
+        Ok(Arc::new(DescribeTableInterpreter { ctx, plan }))
+    }
+}
+
+#[async_trait::async_trait]
+impl Interpreter for DescribeTableInterpreter {
+    fn name(&self) -> &str {
+        "DescribeTableInterpreter"
+    }
+
+    async fn execute(&self) -> Result<SendableDataBlockStream> {
+        let datasource = self.ctx.get_datasource();
+        let database = datasource.get_database(self.plan.db.as_str())?;
+        let table = database.get_table(self.plan.table.as_str())?;
+        let schema = table.schema()?;
+
+        let mut names: Vec<String> = vec![];
+        let mut types: Vec<String> = vec![];
+        let mut nulls: Vec<String> = vec![];
+        for field in schema.fields().iter() {
+            names.push(field.name().to_string());
+            types.push(format!("{:?}", field.data_type()));
+            nulls.push(if field.is_nullable() {
```

Comment:
maybe use `BoolSeries` is better? (true or false)

Reply:
Here reference the Mysql syntax
https://dev.mysql.com/doc/refman/8.0/en/show-columns.html

Reply:
Understand. mysql style is good.